src/common/components/hosts/MassChangeHostnameModal.tsx

```diff
@@ -164,11 +165,13 @@ const MassChangeHostnameForm: React.FC<MassChangeHostnameFormProps> = ({
                 richValidationMessages={HOSTNAME_VALIDATION_MESSAGES}
```

Comment:
The texts in `HOSTNAME_VALIDATION_MESSAGES` and similar constants in the same file are missing translations.